### PR TITLE
update docs/install.md

### PR DESCRIPTION
**Pull request by AldoFusterTurpin**
Minor update on the install.md doc to clarify the steps and provide some links to official OpenShift docs.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -4,12 +4,16 @@
...
-  1. Create a 'hive' namespace.
-  1. In OpenShift web console navigate to Administrator perspective > Operators > OperatorHub.
-  1. Search for “hive” and select the OpenShift Hive operator.
-     1. Select the “alpha” update channel, install to a specific namespace, select the “hive” namespace just created, approval strategy automatic, and press Install.
-     1. You should now have a hive-operator pod running in the hive namespace.
-  1. Create a HiveConfig to trigger the the actual deployment of Hive. (can be done via the web UI with a couple clicks, or with oc apply)
+  1. Create a 'hive' namespace doing
+  `$ oc new-project hive`
+
+  2. Install the Hive Operator:
```
Comment (abutcher) [inline docs/install.md:10]: These bullets should be prefixed with `1.` s.t. the number enumeration is done by markdown. The newline preceding this line will need to be removed for this to work.

File: docs/install.md [hunk 2]
```diff
@@ -4,12 +4,16 @@
...
 Hive is published to [OperatorHub](https://operatorhub.io/operator/hive-operator) weekly and this is the best method to install and use Hive if you do not need to build from source.
 
-  1. Create a 'hive' namespace.
-  1. In OpenShift web console navigate to Administrator perspective > Operators > OperatorHub.
-  1. Search for “hive” and select the OpenShift Hive operator.
-     1. Select the “alpha” update channel, install to a specific namespace, select the “hive” namespace just created, approval strategy automatic, and press Install.
-     1. You should now have a hive-operator pod running in the hive namespace.
-  1. Create a HiveConfig to trigger the the actual deployment of Hive. (can be done via the web UI with a couple clicks, or with oc apply)
+  1. Create a 'hive' namespace doing
+  `$ oc new-project hive`
```
Comment (abutcher) [inline docs/install.md:8]: Not sure if this indentation is correct but wrap the command in triple ticks ``` please.

```suggestion
  1. Create a 'hive' namespace,
      ```
      oc new-project hive
      ```
```

File: docs/install.md [hunk 3]
```diff
@@ -4,12 +4,16 @@
...
+
+  2. Install the Hive Operator:
+  * In the [OpenShift web console](https://docs.openshift.com/container-platform/4.11/web_console/web-console-overview.html), navigate to [Administrator perspective](https://docs.openshift.com/container-platform/4.11/web_console/web-console-overview.html#accessing-the-administrator-perspective_web-console-overview) > Operators > [OperatorHub](https://docs.openshift.com/container-platform/4.11/operators/understanding/olm-understanding-operatorhub.html#olm-operatorhub-overview_olm-understanding-operatorhub).
```
Comment (abutcher) [inline docs/install.md:11]: I think linking to documentation is okay and we do this in multiple places already (`grep -ir "docs.openshift.com" | wc -l` => `11`) but the links are instantly outdated as soon as a new version is released. The docs site supports specifying `latest` as the version in the URL so perhaps that is a good trade off assuming the URL path remains active.

File: docs/install.md [hunk 4]
```diff
@@ -20,6 +24,7 @@ spec:
   logLevel: debug
   targetNamespace: hive
 ```
+  * apply the yaml file doing `$ oc apply -f hive_config.yaml`
```
Comment (abutcher) [inline docs/install.md:27]: Wrap command in ```, indentation may be incorrect.

```suggestion
  * Apply hive_config.yaml,
      ```
      oc apply -f hive_config.yaml
      ```
```

File: docs/install.md [hunk 5]
```diff
@@ -4,12 +4,16 @@
...
+  1. Create a 'hive' namespace doing
+  `$ oc new-project hive`
+
+  2. Install the Hive Operator:
+  * In the [OpenShift web console](https://docs.openshift.com/container-platform/4.11/web_console/web-console-overview.html), navigate to [Administrator perspective](https://docs.openshift.com/container-platform/4.11/web_console/web-console-overview.html#accessing-the-administrator-perspective_web-console-overview) > Operators > [OperatorHub](https://docs.openshift.com/container-platform/4.11/operators/understanding/olm-understanding-operatorhub.html#olm-operatorhub-overview_olm-understanding-operatorhub).
+  * Search for “hive” and select the "Hive for Red Hat OpenShift" operator and click Install.
+  * Select the “alpha” update channel, install to a specific namespace (select the “hive” namespace previously created), approval strategy: automatic, and press Install.
+  * You should now have a hive-operator pod running in the hive namespace.
+  3. Create a HiveConfig to trigger the the actual deployment of Hive.
+  * create a hive_config.yaml file with this content:
```
Comment (abutcher) [inline docs/install.md:16]:
```suggestion
  * Create a hive_config.yaml file with the following content,
```

**Review by abutcher — COMMENTED** [review]
Thanks! Added a few rewording suggestions and a musing on linking to versioned documentation /cc @2uasimojo

**Comment by 2uasimojo** [comment]
> /cc @2uasimojo 

I have looked at this. Agree with your review. For rendering questions, note that you can look at the rendered file by clicking the three dots in the upper right corner of the diff and choosing "View File".

File: docs/install.md (hunk 1 above)
Comment (AldoFusterTurpin) [reply docs/install.md:10]: Done. Thanks!

File: docs/install.md (hunk 2 above)
Comment (AldoFusterTurpin) [reply docs/install.md:8]: Done.

File: docs/install.md (hunk 3 above)
Comment (AldoFusterTurpin) [reply docs/install.md:11]: I wasn't sure how to refer to "latest" in the link. Done. Thanks!

File: docs/install.md (hunk 4 above)
Comment (AldoFusterTurpin) [reply docs/install.md:27]: Done. Thanks!

**Review by abutcher — APPROVED** [review]
Thanks
/lgtm

**Pull request merged**